src/ducks/account/AccountSwitch.jsx

```diff
@@ -437,18 +437,11 @@ AccountSwitch.defaultProps = {
   color: 'default'
 }
 
-const mapStateToProps = (state, ownProps) => {
-  const enhancedState = {
-    ...state,
-    accounts: ownProps.accounts,
-    groups: ownProps.groups
-  }
-  return {
-    virtualGroups: getVirtualGroups(state),
-    filteringDoc: getFilteringDoc(state),
-    filteredAccounts: getFilteredAccounts(enhancedState)
-  }
-}
+const mapStateToProps = state => ({
+  filteringDoc: getFilteringDoc(state),
+  filteredAccounts: getFilteredAccounts(state),
+  virtualGroups: getVirtualGroups(state)
```

Comment:
Why don't we use `createStructuredSelector` here too?

Reply:
We could, I wanted to see if you liked it ;)

Reply:
Ah I already used it in a later commit ;)